package org.hackathon.exception;

public class DuplicateUsernameFoundException extends RuntimeException {
}
